### PR TITLE
[Draft][one-cmds] Add create-quant-dataset option to onecc

### PR DESCRIPTION
**Pull request by Seunghui98**
This commit adds create-quant-dataset option to onecc. 
Add, it contains test cases of onecc.

ONE-DCO-1.0-Signed-off-by: SeungHui Lee <shsh1004.lee@samsung.com>

---
Related issue : https://github.com/Samsung/ONE/issues/12053

### CODE REVIEW

**Comment by Seunghui98** [comment]
@jinevening 
In Draft PR, only `create-quant-dataset` is added in `onecc`, so it can only be used as a onecc command, and not in `cfg`.
Is it necessary to process `create-quant-dataset` in CfgRunner as well?

File: compiler/one-cmds/tests/onecc_059.test [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+trap_err_onexit()
+{
+  echo "${filename_ext} FAILED"
+  exit 255
+}
+
+trap trap_err_onexit ERR
+
+outputfile="output_onecc_one-create-quant-dataset_testdata_001.h5"
```
Comment (jinevening) [inline compiler/one-cmds/tests/onecc_059.test:30]: The output file name is the same with onecc_neg_037.test. Please use a unique output file name for each test.

For example,
```suggestion
outputfile="onecc_059_qdata.h5"
```

File: compiler/one-cmds/tests/onecc_neg_037.test [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  fi
+
+  echo "${filename_ext} FAILED"
+  exit 255
+}
+
+trap trap_err_onexit ERR
+
+inputfile="./wronglist.txt"
+outputfile="output_onecc_one-create-quant-dataset_testdata_001.h5"
```
Comment (jinevening) [inline compiler/one-cmds/tests/onecc_neg_037.test:36]: ditto

File: compiler/one-cmds/tests/onecc_neg_037.test [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    exit 0
+  fi
+
+  echo "${filename_ext} FAILED"
+  exit 255
+}
+
+trap trap_err_onexit ERR
+
+inputfile="./wronglist.txt"
```
Comment (jinevening) [inline compiler/one-cmds/tests/onecc_neg_037.test:35]:
```suggestion
inputfile="./onecc_neg_037_wronglist.txt"
```

File: compiler/one-cmds/onecc [hunk 4]
```diff
@@ -103,6 +106,10 @@ def _get_parser():
     for tool, desc in subtool_list['backend'].items():
         backend_group.add_argument(tool, action='store_true', help=desc)
 
+    tool_group = parser.add_argument_group('run tool for our toolchain')
```
Comment (jinevening) [inline compiler/one-cmds/onecc:109]:
```suggestion
    tool_group = parser.add_argument_group('run tools useful for our toolchain')
```

**Comment by jinevening** [comment]
> Is it necessary to process create-quant-dataset in CfgRunner as well?

I think yes, for consistency with other tools. AFAIK, all cmdline tools can be invoked in cfg file.

**Pull request closed without merging**